app/assets/stylesheets/blacklight/_bookmark.scss

```diff
@@ -18,3 +18,16 @@ label.toggle-bookmark {
 		display: none;
 	}
 }
+
+.toggle-bookmark .blacklight-icons svg {
+  height: 1.25rem;
+  width: 1.25rem;
+  path {
+    fill:none;
+    stroke: var(--bs-primary);
+  }
+}
+
+.toggle-bookmark.checked .blacklight-icons path {
```

Comment:
Can you explain why we're using a css class (`.checked`) rather than the `:checked` pseudo-class?

Reply:
Either will work I figured since we actually attach the class there is made sense.